docs/sources/setup-grafana/configure-security/planning-iam-strategy.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Planning IAM integration strategy
+menuTitle: Planning IAM integration strategy
+description: Considerations and planification before setting up Grafana for the first time.
+aliases:
+  - /docs/mimir/latest/old-architecture/
+  - docs/sources/auth/planning/
+weight: 100
+keywords:
+  - IdP
+  - IAM
+  - Auth
+---
+
+# Planning IAM integration strategy
+
+This section describes the decisions you should make when using an Identity and Access Management (IAM) provider to manage access to Grafana. IAM ensures that users have secure access to sensitive data and [Grafana resources], simplifying user management and authentication.
```

Comment:
Is [Grafana resources] supposed to be a link?

Reply:
Link fixed